src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -2667,6 +2694,45 @@ def cli_cosmosdb_managed_cassandra_datacenter_update(client,
     if backup_storage_customer_key_uri is None:
         backup_storage_customer_key_uri = data_center_resource.properties.backup_storage_customer_key_uri
 
+    is_ldap_properties_none = False
+    if data_center_resource.properties.authentication_method_ldap_properties is None:
+        is_ldap_properties_none = True
+
+    if server_hostname is None and is_ldap_properties_none is False:
```

Comment:
Even if the ldap properties is none, customer should be able to update. Say they have a DC with cassandra auth, and they want to switch to ldap, they should be able to update the ldap properties on the datacenter

Reply:
Yes, from line 2725, the passed ldap values from user will be used to update the ldap settings.